src/shared/ui/paragraph/paragraph.module.scss

```diff
@@ -10,3 +8,18 @@
     line-height: 24px;
   }
 }
+
+.medium {
```

Comment:
I suppose, we have to add `padding` property in all cases (we've discussed it for `Title`)

Reply:
[Resolved](https://github.com/rolling-scopes/site/pull/475/commits/c0158c42b2443716e3e9157717628ee7e2503cb4).

Reply:
We need it only in 2 cases - I suppose, we have to fix it WO additional option